components/project-loader/project-loader.jsx

```diff
@@ -129,6 +134,7 @@ export default React.createClass({
     return (
       <div>
         { this.renderTagHeader() }
+		{ this.renderLearnMoreNotice()}
```

Comment:
Please fix spacing/indentation here and the lines you added above. Make sure your code editor's indentation setting is set to `spaces` instead of `tabs`.